app/client/src/features/events/Questions/AskQuestion/QuestionDialog.tsx

```diff
@@ -15,28 +16,6 @@ export interface Props {
     eventId: string;
 }
 
-export const ASK_QUESTION_MUTATION = graphql`
-    mutation AskQuestionMutation($input: CreateQuestion!) {
```

Comment:
You don't have to remove it, you could just rename it. It should be fine to keep it all in one file. The main point is just the name should match. So here you could just rename it to `QuestionDialogMutation` and `QUESTION_DIALOG_MUTATION` then re-run the relay compiler and update the __generated__ file import to match.

Reply:
Got it, I just updated it with the new names.